package.json

```diff
@@ -29,7 +29,12 @@
     "prebuild": "npm run lint && npm run clean",
     "build": "pixify -n PIXI -o pixi",
     "postbuild": "npm test",
-    "docs": "jsdoc -c scripts/jsdoc.conf.json -R README.md"
+    "docs": "jsdoc -c scripts/jsdoc.conf.json -R README.md",
+    "publish:patch": "npm version patch --no-git-tag-version && npm publish",
+    "publish:minor": "npm version minor --no-git-tag-version && npm publish",
+    "publish:major": "npm version major --no-git-tag-version && npm publish",
+    "prepublish": "npm run build",
```

Comment:
Testing reveals it runs after installing dependencies. From the NPM docs about **prepublish**: `prepublish: Run BEFORE the package is published. (Also run on local npm install without any arguments.)`